scripts/install/opt/seagate/cortx/motr/libexec/motr_cfg.sh

```diff
@@ -179,16 +179,19 @@ do_m0provision_action()
     fi
 
     SALT_OPT=$(salt-call --local grains.get virtual)
+    platform=$(get_platform)
+    echo "Server type is $platform"
 
     ANY_ERR=$(echo $SALT_OPT | grep -i ERROR | wc -l)
     if [ "$ANY_ERR" != "0" ]; then
-        err "Salt command failed."
+        msg "Salt command failed or not available."
         msg "[$SALT_OPT]"
-        die $ERR_CFG_SALT_FAILED
+        #die $ERR_CFG_SALT_FAILED
```

Comment:
Why is this commented out?
Why don't we exit with error?

Reply:
If salt is not available and motr rpm is installed on any physical machine, these changes are applied, see line number 194.